src/sections/Meshery/Meshery-integrations/index.js

```diff
@@ -25,7 +25,7 @@ const Integrations = ({ theme }) => {
               <>
                 <IntegrationsGrid theme={theme} count={13} />
                 <div style={{ marginTop: "20px" }}>
-                  <Button
+                  <Button className="seeAll-btn"
```

Comment:
Please use consistent name of class like see-all-btn or seeAllBtn

Reply:
I have made the necessary changes